### PR TITLE
COST-749: Add materialize views for the storage & database apis.

### PR DESCRIPTION
**Pull request by myersCody**
Added the materialized views for the network and database apis.

### CODE REVIEW

File: koku/reporting/provider/gcp/models.py [hunk 1]
```diff
@@ -785,3 +787,71 @@ class Meta:
...
+
+    id = models.IntegerField(primary_key=True)
+
+    usage_start = models.DateField(null=False)
+
+    usage_end = models.DateField(null=False)
+
+    account_id = models.CharField(max_length=50, null=False)
+
+    usage_amount = models.DecimalField(max_digits=24, decimal_places=9, null=True)
```
Comment (myersCody) [inline koku/reporting/provider/gcp/models.py:813]: I need to update the usage amount in these to match: https://github.com/project-koku/koku/pull/2648

**Review by adberglund — DISMISSED** [review]
Can be a follow up, but don't forget about https://github.com/project-koku/koku/blob/master/koku/api/report/gcp/query_handler.py#L40-L41 and https://github.com/project-koku/koku/blob/master/koku/api/report/gcp/provider_map.py#L258

File: koku/reporting/provider/gcp/models.py (hunk 1 above)
Comment (myersCody) [reply koku/reporting/provider/gcp/models.py:813]: I lied, we use the `usage_in_pricing_units` here.

**Comment by myersCody** [comment]
> Can be a follow up, but don't forget about https://github.com/project-koku/koku/blob/master/koku/api/report/gcp/query_handler.py#L40-L41 and https://github.com/project-koku/koku/blob/master/koku/api/report/gcp/provider_map.py#L258

Whoops! Great Catch @adberglund I will go ahead and add them into this pr.

File: koku/api/report/gcp/query_handler.py [hunk 2]
```diff
@@ -37,8 +37,16 @@ class GCPReportQueryHandler(ReportQueryHandler):
     """Handles report queries and responses for GCP."""
 
     provider = Provider.PROVIDER_GCP
-    network_services = {}  # FIXME: When we start working on networking apis
-    database_services = {}  # FIXME: When we start working on database apis
+
+    network_services = {
```
Comment (adberglund) [inline koku/api/report/gcp/query_handler.py:41]: Will want to match the same strings that are in the materialized view and have the UI update to query with those strings.

**Review by adberglund — APPROVED** [review]

**Pull request merged**